ros-conversion/v2x-ros-conversion/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+
+# Copyright (C) 2020-2024 LEIDOS.
```

Comment:
Since this is a new package, lets update the copyright for the files under this to just be 2024